app/scripts/directives/podMetrics.js

```diff
@@ -358,6 +358,11 @@ angular.module('openshiftConsole')
               // get the unit value if specified
               var unit =  metric.unit || "";
 
+              // A typical metric ID is of the form "pod/<pod-id>/custom/<some-endpoint-info>"
+              // such as "pod/be381d4b-87fc-11e5-b2a3-525400b33d1d/custom/JVM-Heap-Memory-Used".
```

Comment:
OK, seeing the example, I suggest we expose `MetricsService.getMetricInfo()` and use that instead.

https://github.com/openshift/origin-web-console/blob/master/app/scripts/services/metrics.js#L105-L113

@mwringe Do you know if the format we assume in that function always holds?